src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -143,9 +142,10 @@ export class BrowsingContextImpl {
   }
 
   /**
+   * The navigable ID.
    * @see https://html.spec.whatwg.org/multipage/document-sequences.html#navigable
    */
-  get navigableId(): string | undefined {
+  get loaderId(): string | undefined {
```

Comment:
Revert this. BrowsingContext should be thought as the BiDi BrowsingContext this is CDP specific naming. 
They both mean roughly the same thing.

Reply:
I still believe this is mistake.